.github/workflows/buildpack-integration-test.yml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+  pull_request:
+  # Runs every day on 12:00 AM PST
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
nit: this is only on Mondays I think, for daily at midnight we'd want `0 0 * * *`